app/src/test/java/com/x8bit/bitwarden/data/platform/repository/AuthenticatorBridgeRepositoryTest.kt

```diff
@@ -45,7 +45,7 @@ class BridgeRepositoryTest {
     private val fakeAuthDiskSource = FakeAuthDiskSource()
     private val fakeSettingsDiskSource = FakeSettingsDiskSource()
 
-    private val bridgeRepository = BridgeRepositoryImpl(
+    private val bridgeRepository = AuthenticatorBridgeRepositoryImpl(
```

Comment:
Not sure if we also want to update the variable name here as well.

Reply:
I like updating, thanks for noticing this!